utils/asychStorageFunctions.tsx

```diff
@@ -6,7 +6,7 @@ export const storeSignedIn = async (signedIn) => {
     console.log(e);
   }
 };
-export const storeOnboarded = async (onboard) => {
+export const storeOnboarded = async (onboard: "true" | "false") => {
   try {
     await AsyncStorage.setItem("onboard", onboard);
```

Comment:
I think it's cleaner to just have onboard be a boolean here instead of defining a custom string type and then you can do `AsyncStorage.setItem("onboard", onboard.toString())`